--- a/src/ap/hostapd.c
+++ b/src/ap/hostapd.c
@@ -4681,7 +4681,7 @@ static void hostapd_fill_csa_settings(struct hostapd_data *hapd,
 	conf->csa_ecsa_beacon_tx = params->csa_ecsa_beacon_tx;
 	conf->csa_no_ht = params->csa_no_ht;
 
-	hapd->iface->conf->he_mu_edca.he_qos_info &= 0xfff0;
+	//hapd->iface->conf->he_mu_edca.he_qos_info &= 0xfff0;
 }
 
 static int hostapd_setup_csa(struct hostapd_data *hapd,
